static/js/index.js

```diff
@@ -27,6 +27,9 @@ exports.postAceInit = function (hook_name, args, cb) {
     $('#editorcontainerbox').animate({
         'top': '55px',
     }, 500);
+    
```

Comment:
When focussing on the "Toggle autorship colors" and "Download" buttons, voiceover is not reading the text of the buttons. It just announces "button" with the fontawesome character.